openshift_scalability/ci/contents/kitchen-sink-ci.yaml

```diff
@@ -1,47 +1,47 @@
 projects:
-  - num: 5
+  - num: 4
```

Comment:
what does updating to 3.9 have to do with reducing the # of templates?

Reply:
This config file is based on svt/openshift_scalability/master-vert-pv.yaml which had 1 project per app.  We started with 5 in the CI Jenkins Test then scaled back to 4 back in April 2017 during OCP 3.5.x when we realized we were overwhelming the nodes with 5.  Updating that value in svt repo so we can run this test on 3.9 and compare to previous results.

Reply:
@jeremyeder @mffiedler PTAL